k8s/upgrades/0.8.2-0.9.0/jiva/jiva_upgrade_runtask.yaml

```diff
@@ -252,6 +233,9 @@ spec:
     {{- jsonpath .JsonResult "{.items[*].metadata.labels.openebs\\.io/version}" | trim | saveAs "listTargetDeployment.version" .TaskResult | noop -}}
     {{- .TaskResult.listTargetDeployment.version | notFoundErr "unknown openebs version" | saveIf "listTargetDeployment.notFoundErr" .TaskResult | noop -}}
 
+    {{- jsonpath .JsonResult "{.items[*].spec.replicas}" | trim | saveAs "listTargetDeployment.replicaCount" .TaskResult | noop -}}
+    {{- .TaskResult.listTargetDeployment.replicaCount | notFoundErr "unknown no.of replicas in target deployment" | saveIf "listTargetDeployment.notFoundErr" .TaskResult | noop -}}
```

Comment:
verifyErr

Reply:
Please check the message.
notFoundErr is used only when something is not found or string is empty or count is 0

Reply:
Done